tests/features/ui/ui_admin.feature

```diff
@@ -73,3 +73,29 @@ Feature: Admin UI
         | functionaladminpriv | uu_fsw | Change Theme | Social Science |
         | functionaladminpriv | uu_geo | Change Theme | Geo            |
         | functionaladminpriv | uu     | Change Theme | Yoda           |
+
+    Scenario Outline: Admin user previews publication terms
+        Given user <user> is logged in
+        When the user navigates to the admin page
+        And the user adds text <text> to publication terms
+        And the user clicks Preview Terms button
+        Then the added text <text> is shown in the preview window
+
+    Examples:
+        | user                | text           |
+        | functionaladminpriv | TemporaryTerms |
+
+
+    Scenario Outline: Admin user updates publication terms
+        Given user <user> is logged in
+        When the user navigates to the admin page
+        And the user adds text <text> to publication terms
+        And the user clicks Update Terms button
+        And the user navigates to the admin page
+        Then the text <text> is displayed in the publication terms textarea
+        And the user removes the <text> from publication terms
+        And the user clicks Update Terms button
```

Comment:
Here I removed the added text from publication terms for cleaning up, but I wonder is it better to have a separate test to do it?

Reply:
Yeah, that is cleaner. Also when it fails easier to determine where the error is.